spec/models/product_line_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe ProductLine, type: :model do
```

Comment:
I would name this ProductInridient but if you are not adding any attributes to this table there is no need for having this model at all as far as I'm concerned. Rails takes care of joint tables for you

Reply:
But perhaps I'm missing something. What is this model used for?

Reply:
Joint table between product and ingredient. All the guides we read about has many to has many associaton said we needed a joint table

Reply:
Okay, I see that. I would suggest calling this model `ProductIngredient`

Reply:
You could probably make use of a `has_and_belongs_to_many` relationship between `Product` and `Ingredient` and rails would take care of the joint table for you... Check this link for more details
http://guides.rubyonrails.org/association_basics.html#choosing-between-has-many-through-and-has-and-belongs-to-many

Reply:
ok, will take a look at it.